### ISSUE TITLE
Will not load decks/stats on startup

### ISSUE THREAD

**Issue by dfclogik** [Epix37/Hearthstone-Deck-Tracker#630]
I was in the middle of an arena run. I alt+tabbed over to save a deck from hearthpwn.com (I had done this many times before so that's not the issue). All of a sudden my decks disappear from the tracker. They are in the appdata folder but I don't know how to get them back onto my tracker.

**Comment by Epix37** [comment]
Maybe you set a filter? (or the import did) Try `DECKS > QUICK FILTER  >ALL`.

**Comment by dfclogik** [comment]
I tried that too but it wasn't that. It was apparently an issue with the deck save button (chrome extension). I tried saving a deck using the button and everything disappeared.

When I manually saved an already-existing deck under a different name by importing via web (hearthpwn), part of the decks reappeared. I imported a second deck existing deck (renamed), and a few more showed up. But none of the decks had any common filters or anything related
